sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/entity/Student.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * 
+ */
+package com.consoleconnect.mw.sonarqube.sample.entity;
+
+/**
+ * @author dxiong
+ *
+ */
+public class Student {
+
+  
+  public String name() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Document this public method by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sq.ci.consoleconnect.com/coding_rules#rule_key=squid%3AUndocumentedApi)